[FEATURE] `gh` style interface for `mtt export`

### Is your feature request related to a problem? Please describe.

Currently for those of us not using the `ase` interface exports (e.g. eOn) are less than ideal:

```bash
mtt export https://huggingface.co/lab-cosmo/upet/resolve/main/models/pet-mad-s-v1.0.2.ckpt -o model.pt
```

Note that the biggest annoyance here is that the URL one gets by going to the models folder and copying the link (right click) is not valid:

```bash
mtt export https://huggingface.co/lab-cosmo/upet/blob/main/models/pet-omat-xl-v1.0.0.ckpt
[2026-01-28 11:07:06][INFO] - Logging to file is disabled.
[2026-01-28 11:07:06][INFO] - Package version: 2025.12
[2026-01-28 11:07:06][INFO] - Package directory: /home/goswami/Git/Github/epfl/lab-cosmo/pixi_envs/nebmmf/.pixi/envs/eongpu/lib/python3.12/site-packages/metatrain
[2026-01-28 11:07:06][INFO] - Working directory: /home/goswami/Git/Github/epfl/lab-cosmo/pixi_envs/nebmmf/case_studies/static_switchover/results/03_neb/01_hcn/static
[2026-01-28 11:07:06][INFO] - Executed command: mtt export https://huggingface.co/lab-cosmo/upet/blob/main/models/pet-omat-xl-v1.0.0.ckpt
[2026-01-28 11:07:06][ERROR] - If the error message below is unclear, please help us improve it by opening an issue at https://github.com/metatensor/metatrain/issues. When opening the issue, please include the full traceback log from '/home/goswami/Git/Github/epfl/lab-cosmo/pixi_envs/nebmmf/case_studies/static_switchover/results/03_neb/01_hcn/static/error.log'. Thank you!

URL 'https://huggingface.co/lab-cosmo/upet/blob/main/models/pet-omat-xl-v1.0.0.ckpt' has an invalid format for the Hugging Face Hub.
```

Which is a clear enough error but still annoying. 

### Describe the solution you'd like

Since we only have one backend store for these models at the moment we should do a `gh` interface, i.e.

```bash
mtt export labcosmo/upet -b main models/pet-omat-xl-v1.0.0.ckpt
```

Should "do the right thing" (and we can add support for `-t` for tags and other things).
        ^------ i.e. construct the right path to the model file or give a helpful error.

### Describe alternatives you've considered

Use Python (annoying).

### Additional context

_No response_

--- a/docs/src/dev-docs/changelog.rst
+++ b/docs/src/dev-docs/changelog.rst
@@ -44,6 +44,7 @@ Added
 - The PET architecture now features a temperature hyperparameter for the softmax
   operation in attention.
 - The FlashMD architecture added fine-tuning capabilities similar to those of PET.
+- Metatrain now has a simpler CLI interface for HuggingFace models
 
 Changed
 #######

--- a/docs/src/getting-started/checkpoints.rst
+++ b/docs/src/getting-started/checkpoints.rst
@@ -69,6 +69,17 @@ Downloading private HuggingFace models is also supported, by specifying the
 corresponding API token with the ``--token`` flag or the ``HF_TOKEN`` environment
 variable.
 
+When working with HuggingFace, a more concise form may be used.
+
+.. code-block:: bash
+
+    # instead of
+    mtt export https://huggingface.co/lab-cosmo/upet/resolve/main/models/pet-mad-s-v1.0.2.ckpt
+    # we can use
+    mtt export lab-cosmo/upet models/pet-mad-s-v1.0.2.ckpt
+    # or with a revision (branch, tag, commit)
+    mtt export lab-cosmo/pet-mad models/pet-mad-v1.0.2.ckpt --revision=v1.0.2
+
 Keep in mind that a checkpoint (``.ckpt``) is only a temporary file, which can have
 several dependencies and may become unusable if the corresponding architecture is
 updated. In constrast, exported models (``.pt``) act as standalone files. For long-term

--- a/src/metatrain/cli/export.py
+++ b/src/metatrain/cli/export.py
@@ -8,7 +8,7 @@ import torch
 from metatomic.torch import ModelMetadata, is_atomistic_model
 from omegaconf import OmegaConf
 
-from ..utils.io import check_file_extension, load_model
+from ..utils.io import check_file_extension, download_model_from_hf, load_model
 from ..utils.metadata import merge_metadata
 from .formatter import CustomHelpFormatter
 
@@ -36,11 +36,14 @@ def _add_export_model_parser(subparser: argparse._SubParsersAction) -> None:
     parser.add_argument(
         "path",
         type=str,
+        nargs="+",
         help=(
-            "Saved model which should be exported. Path can be either a URL or a "
-            "local file."
+            "Saved model which should be exported. Path can be either a URL, a "
+            "local file, or a Hugging Face Hub identifier followed by the file name "
+            "(e.g. 'metatensor/metatrain-test model.ckpt')."
         ),
     )
+
     parser.add_argument(
         "-o",
         "--output",
@@ -72,6 +75,17 @@ def _add_export_model_parser(subparser: argparse._SubParsersAction) -> None:
         dest="metadata",
         default=None,
         help="Metatdata YAML file to be appended to the model.",
+    )
+    parser.add_argument(
+        "-r",
+        "--revision",
+        "-b",
+        "--branch",
+        dest="revision",
+        type=str,
+        default=None,
+        required=False,
+        help="Revision (branch, tag, or commit) to download from Hugging Face.",
     )
     parser.add_argument(
         "--token",
@@ -107,23 +121,54 @@ def _prepare_export_model_args(args: argparse.Namespace) -> None:
     if args.metadata is not None:
         args.metadata = ModelMetadata(**OmegaConf.load(args.metadata))
 
+    # Handle the nargs='+' for path
+    path_args = args.path
+    if len(path_args) == 1:
+        args.path = path_args[0]
+        args.path_in_repo = None
+    elif len(path_args) == 2:
+        args.path = path_args[0]
+        args.path_in_repo = path_args[1]
+    else:
+        raise ValueError(
+            "Too many arguments provided for 'path'."
+            f" Expected 1 or 2, got {len(path_args)}: {path_args}"
+        )
+
     # only these are needed for `export_model``
-    keys_to_keep = ["path", "output", "extensions", "hf_token", "metadata"]
+    keys_to_keep = [
+        "path",
+        "path_in_repo",
+        "output",
+        "extensions",
+        "hf_token",
+        "metadata",
+        "revision",
+    ]
     original_keys = list(args.__dict__.keys())
 
     for key in original_keys:
         if key not in keys_to_keep:
             args.__dict__.pop(key)
+
+    # Logic to determine default output filename based on input source
     if args.__dict__.get("output") is None:
-        args.__dict__["output"] = Path(args.path).stem + ".pt"
+        if args.path_in_repo is not None:
+            stem = Path(args.path_in_repo).stem
+        else:
+            stem = Path(args.path).stem
+
+        args.__dict__["output"] = stem + ".pt"
 
 
 def export_model(
     path: Union[Path, str],
     output: Union[Path, str],
+    path_in_repo: Optional[str] = None,
     extensions: Union[Path, str] = "extensions/",
     hf_token: Optional[str] = None,
     metadata: Optional[ModelMetadata] = None,
+    revision: Optional[str] = None,
 ) -> None:
     """Export a trained model allowing it to make predictions.
 
@@ -131,13 +176,44 @@ def export_model(
     be saved with a ``.pt`` file ending. If ``path`` does not end with this file
     extensions ``.pt`` will be added and a warning emitted.
 
-    :param path: path to a model file to be exported
+    The user can specify the model in three ways:
+
+    1.  **Local File**:
+
+        .. code-block:: bash
+
+            mtt export model.ckpt
+
+    2.  **Hugging Face Repository** (GitHub-style):
+
+        .. code-block:: bash
+
+            mtt export metatensor/metatrain-test model.ckpt
+
+    3.  **Any URL**:
+
+        .. code-block:: bash
+
+            mtt export https://huggingface.co/metatensor/metatrain-test/resolve/main/model.ckpt
+
+    :param path: path to a model file to be exported, or a Hugging Face repo ID
     :param output: path to save the model
+    :param path_in_repo: path to the model file within the Hugging Face repository
     :param extensions: path to save the extensions
     :param hf_token: HuggingFace API token to download (private) models from HuggingFace
         (optional)
     :param metadata: metadata to be appended to the model
+    :param revision: Revision (branch, tag, or commit) to download from Hugging Face
     """
+    # Resolve Hugging Face repository path if applicable
+    if path_in_repo is not None:
+        logging.info(f"Downloading '{path_in_repo}' from '{path}'...")
+        path = download_model_from_hf(
+            repo_id=str(path),
+            filename=path_in_repo,
+            revision=revision,
+            token=hf_token,
+        )
 
     if Path(output).suffix == ".ckpt":
         checkpoint = torch.load(path, weights_only=False, map_location="cpu")

--- a/src/metatrain/utils/io.py
+++ b/src/metatrain/utils/io.py
@@ -98,21 +98,36 @@ def _hf_hub_download_url(
     revision = unquote(match.group("revision"))
     filename = unquote(match.group("filename"))
 
-    # Extract subfolder if applicable
-    parts = filename.split("/", 1)
-    if len(parts) == 2:
-        subfolder, filename = parts
-    else:
-        subfolder = None
-
     return hf_hub_download(
         repo_id=repo_id,
         filename=filename,
-        subfolder=subfolder,
         cache_dir=cache_dir,
         revision=revision,
         token=hf_token,
         endpoint=endpoint,
+    )
+
+
+def download_model_from_hf(
+    repo_id: str,
+    filename: str,
+    revision: Optional[str] = None,
+    token: Optional[str] = None,
+) -> str:
+    """
+    Download a model file from the Hugging Face Hub.
+
+    :param repo_id: The repository ID (e.g., "metatensor/metatrain-test").
+    :param filename: The filename within the repository (e.g., "model.ckpt").
+    :param revision: The specific revision (branch, tag, or commit hash) to download.
+    :param token: Hugging Face API token for private repositories.
+    :return: The local path to the downloaded file.
+    """
+    return hf_hub_download(
+        repo_id=repo_id,
+        filename=filename,
+        revision=revision,
+        token=token,
     )
 
 

--- a/tests/cli/test_export_model.py
+++ b/tests/cli/test_export_model.py
@@ -255,3 +255,57 @@ def test_export_checkpoint_with_metadata(monkeypatch, tmp_path):
     )
 
     assert f"This is the {model_name} model" in str(model.metadata())
+
+
+def test_huggingface_gh_interface(monkeypatch, tmp_path):
+    """Test that the export cli succeeds when exporting using the
+    GitHub-style interface (repo_id + filename)."""
+    monkeypatch.chdir(tmp_path)
+
+    hf_token = os.getenv("HUGGINGFACE_TOKEN_METATRAIN")
+    if hf_token is None or len(hf_token) == 0:
+        pytest.skip("HuggingFace token not found in environment.")
+
+    command = [
+        "mtt",
+        "export",
+        "metatensor/metatrain-test",
+        "model.ckpt",
+        f"--token={hf_token}",
+    ]
+
+    output = "model.pt"
+
+    subprocess.check_call(command)
+    assert Path(output).is_file()
+
+    # Test if extensions are saved
+    extensions_glob = glob.glob("extensions/")
+    assert len(extensions_glob) == 1
+
+    # Test that the model can be loaded
+    load_model(output, extensions_directory="extensions/")
+
+
+def test_huggingface_gh_interface_revision(monkeypatch, tmp_path):
+    """Test that the export cli succeeds when exporting using the
+    GitHub-style interface with a revision."""
+    monkeypatch.chdir(tmp_path)
+
+    hf_token = os.getenv("HUGGINGFACE_TOKEN_METATRAIN")
+    if hf_token is None or len(hf_token) == 0:
+        pytest.skip("HuggingFace token not found in environment.")
+
+    command = [
+        "mtt",
+        "export",
+        "metatensor/metatrain-test",
+        "model.ckpt",
+        "--revision=main",
+        f"--token={hf_token}",
+    ]
+
+    output = "model.pt"
+
+    subprocess.check_call(command)
+    assert Path(output).is_file()

